Source/Core/DiscIO/VolumeWad.h

```diff
@@ -46,12 +46,12 @@ class CVolumeWAD : public IVolume
 	std::unique_ptr<IBlobReader> m_pReader;
 	u32 m_offset;
 	u32 m_tmd_offset;
-	u32 m_opening_bnr_offset;
-	u32 m_hdr_size;
-	u32 m_cert_size;
-	u32 m_tick_size;
+	u32 m_opening_banner_offset;
```

Comment:
I'm not sure if this should be renamed. The name is based on the filename `opening.bnr`. While that is a banner file, it's not actually named "opening banner".